flagsmith-core.ts

```diff
@@ -439,13 +441,17 @@ const Flagsmith = class {
                                 }
 
                                 if (cachePopulated) { // retrieved flags from local storage
-                                    const shouldFetchFlags = !preventFetch && (!this.cacheOptions.skipAPI||!cachePopulated)
+                                    const shouldFetchFlags = !preventFetch && (
+                                        !this.cacheOptions.skipAPI ||
+                                        !cachePopulated ||
+                                        (cachePopulated && staleCachePopulated)
```

Comment:
Hmm I think this can be tidied up, this is within an ``if(cachePopulated)``, so some of these are redundant ?